docs/running-on-mesos.md

```diff
@@ -211,6 +211,14 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
 </tr>
 <tr>
+  <td><code>spark.mesos.executor.cores</code></td>
+  <td>1.0</td>
+  <td>
+    The setting, which can be a floating point number, controls the number of cores allocated
```

Comment:
Sorry, maybe I should have been more clear, but I meant for this text to go in *in addition* to the paragraph you originally had above it.